ui/lib/core/addon/components/json-editor.hbs

```diff
@@ -27,6 +27,19 @@
               <Icon @name="reload" />
             </button>
           {{/if}}
+          {{#if (and @allowObscure @readOnly)}}
+            {{! For safety we only use obscured values in readonly mode }}
+            <div>
+              <Toggle
+                @name="revealValues"
+                @checked={{this.revealValues}}
+                @size="small"
```

Comment:
The original doesn't have `@size="small"` - is this discrepancy intentional?

Reply:
Yes, in 1.15 `Toggle` does not default to small size so we have to pass it